run/helm/snap/templates/config/_task.tpl

```diff
@@ -0,0 +1,19 @@
+---
+max-failures: 10
```

Comment:
Can you help me understand why there isn't template logic in here and all of it is in the `_helpers.tpl` file? I am not completely sure of the reasoning for doing it this way